### PR TITLE
Branch mon

### PR DESCRIPTION
**Pull request by Santiag0C**
adds percentage_home_wins method

### CODE REVIEW

File: lib/game_statistics_module.rb [hunk 1]
```diff
@@ -11,6 +11,42 @@ def lowest_total_score
...
+  def percentage_home_wins
+    games_won = 0
+    @games.each do |game|
+      if game.home_goals > game.away_goals
+        games_won += 1
+      end
+    end
+    percentage = (games_won.to_f/@games.count) * 100
+    percentage.round(2)
+  end
```
Comment (james-cape) [inline lib/game_statistics_module.rb:35]: This method is already in master (let's remove it)

File: lib/game_statistics_module.rb [hunk 2]
```diff
@@ -11,6 +11,42 @@ def lowest_total_score
...
+  def count_of_games_by_season
+    hash = {}
+    @games.each do |game|
+      if hash.key?(game.season) == false
+        hash[game.season] = 1
+      elsif hash.key?(game.season) == true
+        hash[game.season] += 1
+      end
+    end
+    hash
```
Comment (james-cape) [inline lib/game_statistics_module.rb:50]: This method is already in master (let's remove it)

File: test/game_statistic_test.rb [hunk 3]
```diff
@@ -6,20 +6,24 @@
 require 'pry'
 
 
-class GameStatisticsTest < Minitest::Test
-  def test_highest_total_score
+class StatTrackerTest < Minitest::Test
+  def setup
```
Comment (james-cape) [inline test/game_statistic_test.rb:10]: Let's keep it GameStatisticsTest

File: test/game_statistic_test.rb [hunk 4]
```diff
@@ -6,20 +6,24 @@
...
     game_path = './data/dummy_game.csv'
-    team_path = './data/dummy_teams.csv'
+    team_path = './data/team_info.csv'
```
Comment (james-cape) [inline test/game_statistic_test.rb:12]: Let's get together and set new dummy tests.

Comment (lpile) [reply test/game_statistic_test.rb:12]: Agree

**Review by lpile — APPROVED** [review]
There might be a merge conflict later on. We will sort through it after merge.

**Pull request merged**